sale_quick/models/sale_order.py

```diff
@@ -0,0 +1,64 @@
+# © 2022 Today Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, models
+from odoo.exceptions import ValidationError
+
+
+class SaleOrder(models.Model):
+    _name = "sale.order"
+    _inherit = ["sale.order", "product.mass.addition"]
+
+    def _get_context_add_products(self):
+        return {
+            "search_default_filter_to_sale": 1,
+            "quick_access_rights_sale": 1,
+        }
+
+    def _get_domain_add_products(self):
+        return []
+
+    def add_product(self):
+        self.ensure_one()
+        res = self._common_action_keys()
+        res["context"].update(self._get_context_add_products())
+        domain = self._get_domain_add_products()
+        if domain:
+            res["domain"] = domain
+        commercial = self.partner_id.commercial_partner_id.name
+        res["name"] = "🔙 {} ({})".format(_("Product Variants"), commercial)
```

Comment:
could you remove "🔙 " ? 
or make a translatable text ? 
thanks !

Reply:
I can but i want to mimic the process defined on purchase_quick.
This arrow indicates to get back to the sale order to see what you did.

Any suggestion for a translatable text?

Reply:
BTW, if we change something here, we propagate this change in any *_quick module to keep coherent in the way we want to use the modules

Reply:
I didn't find a way to set a title html tag to emphase/explain the behavior.
Any explanation with text will make the UI complex doesn't it ?
I think this unicode char is self explained : switch back to sale order but other solution can be add : explain it in usage section of the module